app/src/main/java/de/westnordost/streetcomplete/quests/building_levels/AddBuildingLevelsForm.kt

```diff
@@ -31,7 +31,12 @@ class AddBuildingLevelsForm : AbstractQuestFormAnswerFragment<BuildingLevelsAnsw
     private val roofLevels get() = roofLevelsInput?.text?.toString().orEmpty().trim()
 
     private val lastPickedAnswers by lazy {
-        favs.get(javaClass.simpleName).map { it.toBuildingLevelAnswer() }
+        favs.get(javaClass.simpleName)
+        .map { it.toBuildingLevelAnswer() }
+        .sortedWith(
+            compareBy<BuildingLevelsAnswer> { it.levels }.thenBy { it.roofLevels }
```

Comment:
Annoyingly, the compiler can't figure out the type of the `thenBy` without the annotation.

Originally I tried sorting by total levels, then by regular levels (descending), but I found it more confusing to parse than a simple, "What's the first number you're looking for, then what's the second".